### PR TITLE
Add starting point for repository handling

### PR DESCRIPTION
**Pull request by rainemak**

### CODE REVIEW

**Review by llewelld — APPROVED** [review]
This looks like helpful info to share. Just one suggestion would be go clarify that all these commands require root access. LGTM though.

File: Reference/Sailfish_OS_Cheat_Sheet/README.md [hunk 1]
```diff
@@ -350,6 +350,57 @@ systemctl restart ofono
 systemctl-user restart voicecall-manager
 ```
 
+## Repository handling
+
```
Comment (rainemak) [inline Reference/Sailfish_OS_Cheat_Sheet/README.md:354]:
```suggestion
Following commands require you to have root access. First execute
```nosh
devel-su
```
```

**Comment by rainemak** [comment]
> This looks like helpful info to share. Just one suggestion would be go clarify that all these commands require root access. LGTM though.

Added clarification.

File: Reference/Sailfish_OS_Cheat_Sheet/README.md [hunk 2]
```diff
@@ -350,6 +350,62 @@ systemctl restart ofono
...
+All following commands require root access rights
```
Comment (rainemak) [inline Reference/Sailfish_OS_Cheat_Sheet/README.md:355]: Strictly speaking not all but the ones that are altering ssu info... imo it's simpler to put like this.

File: Reference/Sailfish_OS_Cheat_Sheet/README.md [hunk 3]
```diff
@@ -350,6 +350,62 @@ systemctl restart ofono
...
+```nosh
+devel-su
+```
+
+Register the device for R&D services
+```nosh
+ssu r
```
Comment (sledges) [inline Reference/Sailfish_OS_Cheat_Sheet/README.md:362]: Worth indicating this is equivalent to Settings | Enable developer updates, for those who are unfamiliar with terminology.

Comment (llewelld) [reply Reference/Sailfish_OS_Cheat_Sheet/README.md:362]: Thanks, nice changes; still LGTM.

**Pull request merged**